testing_and_setup/compass/utility_scripts/LIGHTparticles/animated_particles.py

```diff
@@ -86,48 +82,51 @@ def build_particle_file(fname_in, fname_outbase):
             if dim == particleTypeStatic:
                 particleData[str(v)] = cleanup(f_in.variables[v][:])
 
-        #print particleData
-
-        # set the new time (just use index for simplicity for now)
-        time = 2*t
-        #time = ''.join(f_in.variables['xtime'][t,0,:]).replace(' ','')
+        if (ignore_xtime) or ('xtime' not in f_in.variables):
+            # Set time to a simple index.
+            time = 2*t
```

Comment:
@pwolfram, this was the default time output in the previous version of the code. Any reason why time is defaulted to `2*t`? Perhaps because 2-day I/O is the default for LIGHT? Maybe this should just be `time=t`.